packages/libs/eda/src/lib/map/analysis/DraggableVisualization.tsx

```diff
@@ -63,7 +63,7 @@ export default function DraggableVisualization({
           confineToParentContainer
           showPanelTitle
           isOpen
-          styleOverrides={{ zIndex: 10 }}
+          styleOverrides={{ zIndex: 10, resize: 'both' }}
```

Comment:
It made more sense to me to take advantage of the `DraggablePanel`'s API that can accept a resize style override and apply it to the panel itself.